guessit/config/options.json

```diff
@@ -503,7 +503,7 @@
         "Read NFO": {"regex": "Read-?NFO"},
         "Converted": {"string": "CONVERT", "tags": "has-neighbor"},
         "Documentary": {"string": ["DOCU", "DOKU"], "tags": "has-neighbor"},
-        "Open Matte": {"string": ["OM", "Open Matte"], "tags": "has-neighbor"},
+        "Open Matte": {"regex": "(?i:OM|Open[ .]+Matte)", "tags": "has-neighbor"},
```

Comment:
I think you could use `-+` instead as rebulk is configured to replace dash with any separator.